reference/configuration/doctrine.rst

```diff
@@ -499,4 +499,135 @@ can be placed directly under ``doctrine.orm`` config level.
 This shortened version is commonly used in other documentation sections.
 Keep in mind that you can't use both syntaxes at the same time.
 
+Custom Mapping
+--------------
+
+It's possible to not rely on the auto mapping and set all the configuration
+on your own.
+
+For example, you can overwrite the configuration directory for a given bundle.
+Keep in mind that the bundle you want to overwrite has to exist.
```

Comment:
i changed your description here, but you're right, this just works if you want to move the configuration folder IN the bundle folder.